homework_classes/src/homework_classes/Circle.java

```diff
@@ -0,0 +1,52 @@
+package homework_classes;
+
+/**
+ * Represents a geometry figure - circle. Contains method to calculate 
+ * area and perimeter of the circle, based on circle radius
+ * @author ajla
+ *
+ */
+public class Circle {
+	private double radius;
+
+	/**
+	 * Constructor
+	 * @param radius
+	 */
+	public Circle(double radius) {
+		try {
+			if (radius > 0) {
+				this.radius = radius;
+			} else {
+				throw new IllegalArgumentException();
+			}
+		} catch (IllegalArgumentException e) {
```

Comment:
You should never throw and catch your own exception in a method.
Since this constructor requires a valid input, best practice would be to throw illegal argument ex. as you did, but do not catch it, let it escape. Also make sure, constructor documentation mentions this use case.